packages/rules-unit-testing/test/impl/discovery.test.ts

```diff
@@ -123,22 +123,6 @@ describe('getEmulatorHostAndPort()', () => {
       expect(result?.host).to.equal('::1');
     });
 
-    it('throws if only host is present', async () => {
```

Comment:
Please don't delete these tests.

Reply:
Sorry, I accidentally bumped them further down the file. I've moved them back now.